### PR TITLE
The Development of 5G protocol standardization

### LINKED ISSUE #1362: The development of 5G protocol standardization.
### Brief Summary: 
This article will describe the LTE evolution and the 5G new technologies such as the NextGen core network and NR which are defined in R15 and R16. This will also cover the various Releases in an attempt to give a good understanding of the development of the 5G protocol standardization.
### Key Takeaways: 
1.	Introduction to 5G protocol standardization
2.	5G networking mode.
3.	Development of uRLLc and mMTC
4.	5G Network security

### References: 
 N/A

### PR DESCRIPTION
**Pull request by RuthAMare**
This fixes #1362 
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/the-development-of-5g-protocol-standardization/index.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
++	Non-standalone/Standalone.
++	Uplink and downlink decoupling.
++ CU-DU high-level segmentation.
+
+*Industry basic design*
++	URLLc (Ultra-Reliable Low Latency Communication).
+
+The implementation of the 5G protocol in Rel-15 underwent a transition through stages, which was split into three stages, as outlined below:
+
+**Early Rel-15 drop:** This stage focused on the architecture option 3, also known as non-standalone NR (NSA NR), which employed the use of an LTE-A system of LTE base stations (called eNB) added to NR base stations (called gNB) and an Evolved Packet Core network (EPC) without any involvement of 5G core network (NGC); this phase of Rel-15 standard was frozen on Dec 2017.
```
Comment (lalith1403) [inline articles/the-development-of-5g-protocol-standardization/index.md:82]: Break this para into smaller sentences. Don't compound more than 2 sentences.

Comment (lalith1403) [reply articles/the-development-of-5g-protocol-standardization/index.md:82]: Make such changes wherever applicable,

Comment (RuthAMare) [reply articles/the-development-of-5g-protocol-standardization/index.md:82]: > Break this para into smaller sentences. Don't compound more than 2 sentences.

Noted, I will do that immediately. Thank you.

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/the-development-of-5g-protocol-standardization/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
++ CU-DU high-level segmentation.
+
+*Industry basic design*
++	URLLc (Ultra-Reliable Low Latency Communication).
+
+The implementation of the 5G protocol in Rel-15 underwent a transition through stages, which was split into three stages, as outlined below:
+
+**Early Rel-15 drop:** This stage focused on the third architecture option, also known as non-standalone NR (NSA NR). It employed the use of an LTE-A system of LTE base stations (called eNB) added to NR base stations (called gNB) and an Evolved Packet Core network (EPC) without any involvement of 5G core network (NGC). This phase of the Rel-15 standard was frozen in Dec 2017.
+
+<<<<<<< HEAD
```
Comment (lalith1403) [inline articles/the-development-of-5g-protocol-standardization/index.md:84]: These strings appear in between. Pull the changes made by me, and then push the changes on your end. This is due to an error in using Git on your local editor.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Technically this article is sound. Congratulations on a well-written article. Change the git strings appearing in the file, and we can proceed to the final round of review,

**Comment by hectorkambow** [comment]
@lalith1403  

If we can get a status on this PR as well 👍

**Comment by lalith1403** [comment]
@hectorkambow this article is ready for the final review. Was waiting for a notification from the author. :)

**Comment by RuthAMare** [comment]
Sorry about the notification @lalith1403...seems i overlooked it and failed to provide any.

**Pull request merged**